### PR TITLE
Ovidio Ean validation

### PR DESCRIPTION
**Pull request by OvidioMiranda**
The code for EAN Validate

### CODE REVIEW

File: src/main/java/org/fundacionjala/coding/marcoslara/spinwords/SpinWords.java [hunk 1]
```diff
@@ -1,32 +0,0 @@
-package org.fundacionjala.coding.marcoslara.spinwords;
```
Comment (carledriss) [inline src/main/java/org/fundacionjala/coding/marcoslara/spinwords/SpinWords.java:1]: Why are you deleting this code?
you need to keep it

File: src/test/java/org/fundacionjala/coding/marcoslara/spinwords/SpinWordsTest.java [hunk 2]
```diff
@@ -1,65 +0,0 @@
-package org.fundacionjala.coding.marcoslara.spinwords;
```
Comment (carledriss) [inline src/test/java/org/fundacionjala/coding/marcoslara/spinwords/SpinWordsTest.java:1]: Same as above

Comment (OvidioMiranda) [reply src/test/java/org/fundacionjala/coding/marcoslara/spinwords/SpinWordsTest.java:1]: How do I fix that now?
Or created a new branch and eliminated this one.

Comment (carledriss) [reply src/test/java/org/fundacionjala/coding/marcoslara/spinwords/SpinWordsTest.java:1]: Do you remember how to apply git reset ?
1. git reset HEAD~6
2. git commit again
3. git push --force

**Pull request closed without merging**